src/commands/lambda/flare.ts

```diff
@@ -1,14 +1,239 @@
+      errorFound = true
+    }
+
+    // Validate Datadog API key
+    this.apiKey = process.env[CI_API_KEY_ENV_VAR] ?? process.env[API_KEY_ENV_VAR]
+    if (this.apiKey === undefined) {
+      this.context.stderr.write(
+        commonRenderer.renderError(
+          'No Datadog API key specified. Set an API key with the DATADOG_API_KEY environment variable.'
+        )
+      )
+      errorFound = true
+    }
+
+    // Validate case ID
+    if (this.caseId === undefined) {
+      this.context.stderr.write(commonRenderer.renderError('No case ID specified. [-c,--case-id]'))
+      errorFound = true
+    }
+
+    // Validate email
+    if (this.email === undefined) {
+      this.context.stderr.write(commonRenderer.renderError('No email specified. [-e,--email]'))
+      errorFound = true
+    }
+
+    if (errorFound) {
+      return 1
+    }
+
+    // Get AWS credentials
+    this.context.stdout.write('\n🔑 Getting AWS credentials...\n')
+    try {
+      this.credentials = await getAWSCredentials()
+    } catch (err) {
+      this.context.stderr.write(commonRenderer.renderError(err))
+
+      return 1
+    }
+    if (this.credentials === undefined) {
+      this.context.stdout.write(commonRenderer.renderNoAWSCredentialsFound())
+      try {
+        await requestAWSCredentials()
+      } catch (err) {
+        this.context.stderr.write(commonRenderer.renderError(err))
+
+        return 1
+      }
+    }
+
+    // Get Lambda function configuration
+    this.context.stdout.write('\n🔍 Getting Lambda function configuration...\n')
+    const lambdaClientConfig: LambdaClientConfig = {
+      region,
+      credentials: this.credentials,
+    }
+    const lambdaClient = new LambdaClient(lambdaClientConfig)
+    let config
+    try {
+      config = await getLambdaFunctionConfig(lambdaClient, this.functionName ?? '')
+    } catch (err) {
+      this.context.stderr.write(
+        commonRenderer.renderError(`Unable to get Lambda function configuration: ${err.message}`)
+      )
+
+      return 1
+    }
+
+    // Print config
+    const configStrColored = util.inspect(config, false, undefined, true)
+    this.context.stdout.write(`\n${configStrColored}\n`)
+    if (this.isDryRun) {
+      this.context.stdout.write('\n🚫 The configuration was not sent as it was executed in dry run mode.\n')
+
+      return 0
+    }
+
+    // Send data to Datadog
+    const folderPath = path.join(process.cwd(), FLARE_OUTPUT_DIRECTORY)
+    const filePath = path.join(folderPath, FUNCTION_CONFIG_FILE_NAME)
+    const zipPath = path.join(folderPath, ZIP_FILE_NAME)
+    const configStrUncolored = JSON.stringify(config, undefined, 2)
+    try {
+      await this.writeFile(folderPath, filePath, configStrUncolored)
+      await this.zipContents(filePath, zipPath)
+      this.context.stdout.write('\n🚀 Sending to Datadog Support...\n')
+      await this.sendToDatadog(zipPath)
+      this.deleteFolder(folderPath)
```

Comment:
This should be something like
```suggestion
      this.context.stderr.write(renderError(err.message))
```